README.md

```diff
@@ -1,70 +1,51 @@
-# Getting Started with Create React App
+Math magician
```

Comment:
Optional

- In order to have a great README file, there isn't a template that is cooler and more professional than [Microvers's one](https://github.com/microverseinc/readme-template)